person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def rent_book(person, date)
+    Rental.new(date, person, self)
```

Comment:
- [ ] We already have access to the `person` object through `self`, so what we need is `book`, not `person`.

- [ ] Kindly take note of the arrangement for the parameters.

  ```
  rent_book(book, date)
    Rental.new(date, book, self)
  ```

Reply:
Thank you for the tip.